### PR TITLE
LPS-178248 | Test Fix | Master

### PR DESCRIPTION
**Pull request by JerryAdrianoo**
**Ticket :**

> [LPS-178248](https://issues.liferay.com/browse/LPS-178248)

### CODE REVIEW

**Comment by JerryAdrianoo** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/PagesConfiguration.path [hunk 1]
```diff
@@ -37,7 +37,7 @@
 </tr>
 <tr>
 	<td>CONTENT_CHOOSE_CONTENT_CONTENT_CHECKBOX</td>
```
Comment (Luizcarlosqueiroz) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/PagesConfiguration.path:39]: This path is used by other macros. Did you check if these changes won't change the behavior for other tests?

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/PagesConfiguration.path [hunk 2]
```diff
@@ -52,7 +52,7 @@
 </tr>
 <tr>
 	<td>PAGES_ALL_PRIVATE_PAGES_CHECKBOX</td>
-	<td>//span[contains(.,'Private Pages')]//input[contains(@type,'checkbox')]</td>
+	<td>//span[contains(.,'Private Pages')]//input[contains(@type,'checkbox')] | //span[contains(.,'Staging Test Page')]//input[contains(@type,'checkbox')]</td>
```
Comment (Luizcarlosqueiroz) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/PagesConfiguration.path:55]: This path is supposed to be only for private pages. I was unable to find where is being used, but in theory, we should use another macro for a Staging page.

**Review by Luizcarlosqueiroz — CHANGES_REQUESTED** [review]
Hey @JerryAdrianoo !  Could you check my comments? Thanks!!

**Review by Luizcarlosqueiroz — APPROVED** [review]
LGTM! :tada:

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**